### PR TITLE
fix: unit test as-vending

### PR DESCRIPTION
**Pull request by joshua-silverio**

### CODE REVIEW

File: as-vending/Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ stop:
 	docker rm -f $(MICROSERVICE):latest
 
 test: lint
-	go test -test.v -cover ./...
+	go test -test.v -cover -count=1 -race ./...
```
Comment (lenny-intel) [inline as-vending/Makefile:37]: Why `-count=1` ?

Comment (joshua-silverio) [reply as-vending/Makefile:37]: we were seeing some caching issues so @jim-wang-intel suggested using -count=1 to clear cache

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by lenny-intel — COMMENTED** [review]
LGTM

**Review by brian-intel — APPROVED** [review]
LGTM

**Review by marcpfuller — APPROVED** [review]

**Review by vli11 — APPROVED** [review]
LGTM

**Pull request merged**